app/invocation/invocation_action_card.tsx

```diff
@@ -485,6 +502,7 @@ export default class InvocationActionCardComponent extends React.Component<Props
         timestampToDate(metadata.workerStartTimestamp).getTime();
       timingDescription = (
         <div>
+          <div className="metadata-title">Timing</div>
```

Comment:
Can you revert all of the changes from this line until the end of the file? These changes might be worth making but seem unrelated to the PR.